cmd/admin_cmd.go

```diff
@@ -115,7 +115,7 @@ func adminDeleteUser(globalConfig *conf.GlobalConfiguration, config *conf.Config
 
 	user, err := db.FindUserByEmailAndAudience(instanceID, args[0], getAudience(config))
 	if err != nil {
-		user, err = db.FindUserByID(args[0])
+		user, err = db.FindUserByInstanceIDAndID(instanceID, args[0])
```

Comment:
User IDs are globally unique, which is why I didn't have the `instance_id` here.

Reply:
I understand that, but by not checking the InstanceID you are giving admins in other instances access to any user.

Reply:
Ah that makes sense. 👍